include/config/config_debug_controls.h

```diff
@@ -0,0 +1,72 @@
+#ifndef CONFIG_DEBUG_CONTROLS_H
+#define CONFIG_DEBUG_CONTROLS_H
+
+/**********************
+ * CONTROLLER MAPPING *
+ **********************/
+
+/**
+ * Change the debug features' controls
+ */
+
+// -------------------------------------------
+
+/**** [NO CLIP CONTROLS] ****/
+// Controller port to use
+#define NOCLIP_CONTROLLER_PORT CONTROLLER_PORT_1
```

Comment:
This is a great idea here! I actually hope that at some point we can do the same thing for z_player as a whole, so we can easily remap the buttons for different actions.

Reply:
yea I wanted to give modders an easy way to change every controls and controller port, there's documentation on TCRF (which helped me a lot) but it's better if you can customise everything imo

also I try my best at making OoT accessible for everyone 😄